dist/info/3dengine_libretro.info

```diff
@@ -9,3 +9,4 @@ license = "GPLv3"
 permissions = ""
 display_version = "v1"
 supports_no_game = "false"
+notes = "(!) 3DEngine does not use any BIOS files."
```

Comment:
People are likely to care about `Firmware(s):`, `BIOS supported? (Yes or no)`, `Supported extensions:` more than `Categories`, `Authors`, `License`.

People probably like seeing `System name, System manufacturer`.

Also, `Core label` often display `Core name`'s value too making `Core name` somewhat redundant. It's taking up a line just same.

You say it's redundant. I don't see that same information anywhere else on RetroArch. I will make a removal commit.